draft-ietf-quic-transport.md

```diff
@@ -2701,10 +2701,10 @@ frames. -->
 {: #packet-frames title="QUIC Payload"}
 
 QUIC payloads MUST contain at least one frame, and MAY contain multiple frames
-and multiple frame types.
+and multiple frame types.  Frames MUST fit within a single QUIC packet and MUST
```

Comment:
This last bit is true, but it's not just a MUST, it seems impossible given the framing we have.  I'm wondering if a can not is more appropriate, given it's essentially impossible?

But I guess this is pre-existing normative text, so in the interest in keeping this PR editorial, this is fine by me.